extensions/dependencies/codemirror.html

```diff
@@ -1,12 +1,12 @@
-<script src="/third_party/static/code-mirror-3.19.0/lib/codemirror.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/coffeescript/coffeescript.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/javascript/javascript.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/lua/lua.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/python/python.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/ruby/ruby.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/scheme/scheme.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/mode/yaml/yaml.js"></script>
-<script src="/third_party/static/ui-codemirror-0.1.2/src/ui-codemirror.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/lib/codemirror.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/coffeescript/coffeescript.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/javascript/javascript.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/lua/lua.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/python/python.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/ruby/ruby.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/scheme/scheme.js"></script>
+<script src="/third_party/static/code-mirror-5.17.0/mode/yaml/yaml.js"></script>
+<script src="/third_party/static/ui-codemirror-0.3.0/src/ui-codemirror.js"></script>
 
-<script src="/third_party/static/code-mirror-3.19.0/addon/merge/dep/diff_match_patch.js"></script>
-<script src="/third_party/static/code-mirror-3.19.0/addon/merge/merge.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/diff_match_patch/20121119/diff_match_patch.js"></script>
```

Comment:
No, serve from third_party/ please. Download it if necessary.

Reply:
Done Thanks!